pyomo/contrib/incidence_analysis/config.py

```diff
@@ -0,0 +1,49 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2022
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+"""Configuration options for incidence graph generation
+"""
+
+import enum
+from pyomo.common.config import ConfigBlock, ConfigValue, InEnum
+
+
+class IncidenceMethod(enum.Enum):
+    identify_variables = 0
+    standard_repn = 1
+
+
+IncidenceConfig = ConfigBlock()
```

Comment:
Technically `ConfigDict` is preferred over `ConfigBlock`

Reply:
Changed